src/core/server/saved_objects/migrations/kibana/kibana_migrator.ts

```diff
@@ -109,12 +124,19 @@ export class KibanaMigrator {
     Array<{ status: string }>
   > {
     if (this.migrationResult === undefined || rerun) {
+      this.status$.next({ status: 'running' });
       this.migrationResult = this.runMigrationsInternal();
     }
```

Comment:
I'm mixed about the fact to re-emit `running` status in case of `runMigrations({ rerun: true})`. OTOH, this should only be used for FTR tests, so I guess that's alright.